app/models/BasicPokemonDetail.js

```diff
@@ -9,6 +9,182 @@ let BasicPokemonDetailSchema = new mongoose.Schema({
     },
     name: {
         type: String
+    },
+    gender: {
+        /*male: {
```

Comment:
I think you can store gender as a boolean value, for e.g. 1 for male and 2 for female.